### ISSUE TITLE
MPRIS player not startet until the first song starts playing, even with `Continue where you left off` enabled

### ISSUE THREAD

**Issue by cwrau** [ytmdesktop/ytmdesktop#1300]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

Currently, when YTMD starts it loads the playlist and song where I left off but requires me to start the playback because I set it to `Pause on application launch`, which is expected.

But I cannot press the `media-play button` (XF86AudioPlay) to start this, as YTMD hasn't created an MPRIS player yet, only after I manually go to the UI and click play I'm able to use the media buttons.

### Expected Behavior

That YTMD would create an MPRIS player as soon as there is a song in the queue

### Steps To Reproduce

1. Enable `Continue where you left of` and `Pause on application launch`
2. Start some songs
3. Restart the application
4. Press the play/pause key on your keyboard
5. observe YTMD not starting playback

### YTMDesktop

2.0.0

### OS

Arch Linux

### OS Version

latest

### Arch

x64

### Installation way

Downloaded the .deb file, extracted it and used the included binary, see https://aur.archlinux.org/packages/ytmdesktop-bin#comment-955354

**Comment by Alipoodle** [comment]
This is sounding like an extension of #1165 and also just how MPRIS works in native Electron 🤔

**Comment by cwrau** [comment]
> This is sounding like an extension of #1165 and also just how MPRIS works in native Electron 🤔
> 
> The feature was changed to work better via using the `Requires user input` to fix this issue:
> [NovusTheoryz's Comment](https://github.com/ytmdesktop/ytmdesktop/issues/1165#issuecomment-1724477545)
> > Right before we released v2.0.0-rc.1 we modified the functionality to utilize autoplay blocking because it provided a better UX. The previous one would try to pause the song as quickly as possible when it played which often ended up with the song playing for 1 second

Dayum, I missed that issue 😅
Then we can close this one an track it in the other one, sorry!

**Comment by Alipoodle** [comment]
Duplicate of #1165  

Session information for all Media stuff likely hasn't started yet to advertise you can play.
